src/metaswitch/clearwater/cluster_manager/constants.py

```diff
@@ -60,3 +60,5 @@
 LEAVING_CONFIG_CHANGED = "leaving, config changed"
 FINISHED = "finished"
 ERROR = "error"
+
```

Comment:
Deliberate - the DELETE_ME state is kind of a pseudo-state and different from the others. I'll comment to that effect.